modules/data-entry/src/main/frontend/src/questionnaire/QuestionnaireStyle.jsx

```diff
@@ -233,6 +233,74 @@ const questionnaireStyle = theme => ({
         },
       },
     },
+    timelineContent: {
+        padding: theme.spacing(1,3),
+    },
+    timelinePaper: {
+        padding: theme.spacing(1,2),
+    },
+    timelineAncestor: {
+        opacity: 0.3,
+        "&:hover": {
+          opacity: 1,
+        },
+    },
+    timelineDate: {
+        marginTop: "-5px",
```

Comment:
We might be able to obtain the same effect without hardcoding pixel sizes, like this:
```suggestion
        lineHeight: 1em,
```